rpm_spec/qubes-video-companion.spec.in

```diff
@@ -37,6 +37,9 @@ BuildArch:      noarch
 Requires:       gstreamer1-plugins-good
 Requires:       python3
 Requires:       v4l-utils
+Requires:       libayatana-appindicator-gtk3
+Requires:       (dunst or notification-daemon)
```

Comment:
Why this particular combination of `Requires`?  `xfce4-notifyd` provides `desktop-notification-daemon` on at least Fedora 39.

Reply:
The first reason is `dunst` with `libnotify` is very lightweight, the second reason is that I didn't know about `xfce4-notifyd`. Should I use `xfce4-notifyd` them?

Reply:
I suggest using the `desktop-notification-daemon` meta-package on Fedora, and whichever meta-package Debian uses (if any).